tests/ti_deps/deps/test_pool_slots_available_dep.py

```diff
@@ -40,22 +40,22 @@ def tearDown(self):
     @patch('airflow.models.Pool.open_slots', return_value=0)
     # pylint: disable=unused-argument
     def test_pooled_task_reached_concurrency(self, mock_open_slots):
-        ti = Mock(pool='test_pool')
+        ti = Mock(pool='test_pool', pool_capacity=1)
         self.assertFalse(PoolSlotsAvailableDep().is_met(ti=ti))
 
     @patch('airflow.models.Pool.open_slots', return_value=1)
     # pylint: disable=unused-argument
     def test_pooled_task_pass(self, mock_open_slots):
-        ti = Mock(pool='test_pool')
+        ti = Mock(pool='test_pool', pool_capacity=1)
```

Comment:
Isn't 1 the default, meaning most of these changes in tests aren't needed?

Reply:
removed pool_capacity from most of the places except few where it is absolutely required.